mesheryctl/internal/cli/root/system/start.go

```diff
@@ -193,7 +195,27 @@ func start() error {
 		}
 
 		var endpoint meshkitutils.HostPort
-		endpoint.Address = "http://localhost"
+
+		userResponse := false
+		if utils.SilentFlag {
+			userResponse = true
+		} else {
+			// ask user for confirmation
+			userResponse = utils.AskForConfirmation("The endpoint address will be changed to localhost. Are you sure you want to continue?")
+		}
+
+		if userResponse {
+			endpoint.Address = utils.EndpointProtocol + "://localhost"
```

Comment:
@metonymic-smokey does this mean that the endpoint is localhost always in a docker deployment?

Reply:
@kumarabd no, not always.
It is changed to `localhost` based on the user's confirmation.